browser-client.cpp

```diff
@@ -373,7 +373,11 @@ void BrowserClient::UpdateExtraTexture()
 
 void BrowserClient::OnAcceleratedPaint(CefRefPtr<CefBrowser>,
 				       PaintElementType type, const RectList &,
+#if CHROME_VERSION_BUILD >= 6367
+				       const CefAcceleratedPaintInfo &info)
+#else
 				       void *shared_handle)
+#endif
 {
 	if (type != PET_VIEW) {
 		// TODO Overlay texture on top of bs->texture
```

Comment:
Also left a word here. This should be removed >= 6367 as it has no guarantee on both macOS and Windows that same handle = same content. On Linux this handle is no more valid because of that complex native pixmap structure.

```
#ifndef _WIN32
	if (shared_handle == bs->last_handle)
		return;
#endif
```

Reply:
As with the other one, is this a theoretical or confirmed problem?

Reply:
In theory this should not be an issue on macOS as textures backed by IOSurfaces are "live" (as in: When whatever the IOSurface references is updated, the texture's visible contents are updated as well).

So if the handle hasn't changed, but the associated texture contents have, it would be reflected by OBS' own texture.

In my tests this was used by the old shared texture implementation heavily (we were able to jump out early because the IOSurface hadn't changed) but in my tests it never seems to happen in the new implementation (the IOSurface is always different, which suggests that every frame is represented by a new device texture?).

Reply:
I have left this code path untouched and version gated to pre-patch, and verified it works on all platforms as-is.